README.md

```diff
@@ -1 +1,57 @@
-# Personal-Portfolio
\ No newline at end of file
+Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+👥 Authors
+🔭 Future Features
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ
+
+Portfolio
```

Comment:
- [ ] Great job using the correct template to document your project! 💯 However, please consider keeping the markdown syntax to keep the structure, at the moment your README collapsed and the content doesn't have any format:
![image](https://user-images.githubusercontent.com/77069011/205150787-430ce4b6-d3d1-47ea-aa90-8b93dcb0ca61.png)

Please, consider going back to the [template](https://github.com/microverseinc/readme-template), copying and pasting it, and customizing it with your project information! 🤓 🛠️